packages/dataviews/src/types.ts

```diff
@@ -689,10 +689,12 @@ export type NormalizedRegularLayout = {
 export type PanelLayout = {
 	type: 'panel';
 	labelPosition?: LabelPosition;
+	panelType?: 'dropdown' | 'modal';
```

Comment:
For naming, what do you think of renaming this prop along the lines of `openAs` instead? So many "type" things that the meaning dilutes, and it all becomes confusing.